pypit/spectrographs/keck_nires.py

```diff
@@ -14,18 +14,42 @@
 
 from pypit import msgs
 from pypit import ardebug as debugger
-from. import spectroclass
+from pypit.par.pypitpar import DetectorPar
+from pypit.spectrographs import spectrograph
+from pypit import telescopes
 
-class KeckNIRESpectrograph(spectroclass.Spectrograph):
+# TODO: KeckNIRESSPectrograph instead (i.e. two SS) ?
+class KeckNIRESpectrograph(spectrograph.Spectrograph):
```

Comment:
Or... `KeckNiresSpectrograph`?

Reply:
Yeah, I agree with you.  We have a somewhat mixed convention at the moment, though.  For example, we have `KeckDEIMOSSpectrograph` and `KeckNIRSPECSpectrograph`, but also `WhtIsisSpectrograph`...  We should discuss a convention and then change these appropriately.  Sorry, I keep doing this, but I'm punting this one to a later PR, as well.